actions/add-review-url/action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Add Review Url'
+description: 'Add Review Url as a comment on PR'
+author: ''
+inputs:  
+  repo-token:  # GitHub Auth Token
+    description: 'repo token'

```

Comment:
We need full sentences for the descriptions.

Reply:
Updated.